### ISSUE TITLE
pretrained lightcnn 4 model

### ISSUE THREAD

**Issue by stormyoung** [SauravRai/Special-Project-Face-Recognition-with-Contrastive-Convolution#1]
Hello, Deer SauravRai,
    I did not find pretrained lightcnn 4 model on the Internet, could you please show the download link for me ?
    By the way, Is there a big difference in performance if do not use pretrained model?
    Thank you! hope for your reply.

**Comment by SauravRai** [comment]
Hi Stormyoung,
                 Sorry for the late reply. Unfortunately, there is no pre-trained lightcnn 4 model in pytorch framework. It is present in caffe framework. I have actually converted it from caffee framework before using in my implementation. If you are not able to do it kindly share your email id, I will forward it in your drive.
If you need any more clarification kindly forward your questions. I will be more than happy to help you.

with regards,
Saurav

**Comment by stormyoung** [comment]
Hi SauravRai,
Thank you for your kindly reply. My email address is: 627103027@qq.com. Thank you for sending pretrained model to me.
Actually, I use this code and train on CASIA, but loss is not convergence. I do not know why, Maybe I can try the pretrained model.

best regards,
Stormyoung
